packages/vue/src/index.ts

```diff
@@ -86,9 +110,16 @@ const Frontegg: PluginObject<PluginOptions> | any = {
     };
 
     const checkIfPluginsLoaded = () => {
-      const _fronteggLoaded = (Vue.fronteggPlugins || []).reduce((loaded: boolean, plugin) => loaded && !plugin.loading, true);
+      const _fronteggLoaded = (Vue.fronteggPlugins || []).reduce(
```

Comment:
This logic should be removed and should wait for FronteggApp loading state.
It's old logic related to the multiple plugins react-auth, react-audits, react-core..

Reply:
lets create separate ticket for that

Reply:
OK